Cargo.toml

```diff
@@ -74,7 +74,9 @@ features = [
     "RequestInit",
     "RequestMode",
     "Response",
-    "Window",
+    
+    # Do not add Window since we want repc to work in a Worker.
```

Comment:
I don't think we'll need the comment here since presumably we'll have tests exercising it in sw which will fail...